libraries/AP_GyroFFT/AP_GyroFFT.cpp

```diff
@@ -655,7 +655,7 @@ float AP_GyroFFT::get_slewed_noise_center_freq_hz(FrequencyPeak peak, uint8_t ax
 {
     uint32_t now = AP_HAL::micros();
     const float slew = MIN(1.0f, (now - _global_state._last_output_us[axis])
-        * (_fft_sampling_rate_hz / _samples_per_frame) / 1e6f);
+    * (static_cast<float>(_fft_sampling_rate_hz) / static_cast<float>(_samples_per_frame)) / 1e6f);
```

Comment:
need to put back the whitespace

Reply:
.... there's also two divisions here rather than a multiple and two divisions :-) That's pre-existing.

Reply:
done